notary_now_api/views.py

```diff
@@ -43,10 +40,10 @@ def notary_verify(request, pk):
 
 @csrf_exempt
 def appointments(request, pk):
-    notary = Notary.objects.filter(user_id=pk)[0]
+    notary = Notary.objects.filter(user_id=pk)
     if notary:
         if request.method == 'GET':
-            appointments = Appointment.objects.filter(notary_id=notary.id)
+            appointments = Appointment.objects.filter(notary_id=pk).order_by('id')
```

Comment:
Looks great. I think filtering by PK is a great idea. Just removes one less concern.